packages/plugins/DecentralizedSearch/src/SNSAdaptor/SearchResultInspector.tsx

```diff
@@ -0,0 +1,152 @@
+        NetworkPluginID.PLUGIN_EVM,
+        props.keyword,
+        [ChainId.Mainnet, ChainId.BSC, ChainId.Matic],
+    )
+
+    const ENS_SearchResult = ENS_Plugin!.SearchResultContent?.UI?.Content!
+    const [isLoading, setLoading] = useState(true)
+    const [isHiddenAll, setHiddenAll] = useState(true)
+    const [isEmpty, setEmpty] = useState(false)
+    const [isError, setError] = useState(false)
+    const socialAccount = {
+        pluginID: NetworkPluginID.PLUGIN_EVM,
+        address: ensRef.current?.reversedAddress ?? '',
+        label: ensRef.current?.domain ?? '',
+        supportedAddressTypes: [SocialAddressType.ENS],
+    }
+    const translate = usePluginI18NField()
+    const activatedPlugins = useActivatedPluginsSNSAdaptor('any')
+    const displayPlugins = useAvailablePlugins(activatedPlugins, (plugins) => {
+        return plugins
+            .flatMap((x) => x.ProfileCardTabs?.map((y) => ({ ...y, pluginID: x.ID })) ?? EMPTY_LIST)
+            .filter((x) => {
+                return x.Utils?.shouldDisplay?.(undefined, socialAccount) ?? true
+            })
+            .sort((a, z) => {
+                // order those tabs from collectible first
+                if (a.pluginID === PluginID.Collectible) return -1
+                if (z.pluginID === PluginID.Collectible) return 1
+
+                return a.priority - z.priority
+            })
+    })
+    const tabs = displayPlugins.map((x) => ({
+        id: x.ID,
+        label: typeof x.label === 'string' ? x.label : translate(x.pluginID, x.label),
+    }))
+    const [currentTab, onChange] = useTabs(first(tabs)?.id ?? PluginID.Collectible, ...tabs.map((tab) => tab.id))
+
+    const component = useMemo(() => {
+        const Component = getProfileTabContent(currentTab)
+        if (!Component) return null
+
+        return <Component identity={undefined} socialAccount={socialAccount} />
+    }, [currentTab, socialAccount])
+
+    useEffect(() => {
+        setLoading(!ensRef.current || ensRef.current?.isLoading)
+        setHiddenAll(Boolean(ensRef.current && ensRef.current?.reversedAddress === undefined) || isLoadingAddressType)
+        setEmpty(
+            Boolean(
+                ensRef.current &&
+                    (!ensRef.current?.reversedAddress || !ensRef.current?.tokenId) &&
+                    !ensRef.current?.isLoading,
+            ),
+        )
+        setError(Boolean(ensRef.current?.isError))
+    }, [ensRef.current, isLoadingAddressType])
+
+    if (
+        addressType === AddressType.Contract ||
+        isLoadingAddressType ||
+        (!addressType && ensRef.current?.reversedAddress && !Others?.isValidDomain?.(props.keyword))
+    )
+        return null
+
+    return (
+        <Hidden hidden={isHiddenAll}>
+            <DecentralizedSearchPostExtraInfoWrapper>
+                <Hidden hidden={isLoading || isEmpty || isError}>
+                    <ENS_SearchResult
```

Comment:
Better avoid dead-coded to ENS result. Any plugin can define itself as a search result.

Reply:
It is defined by ENS plugin.